plugins/panels/model_evaluation/utils.py

```
@@ -0,0 +1,104 @@
+import numpy as np
+from fiftyone.operators.cache.serialization import (
+    auto_serialize,
+    auto_deserialize,
+)
+
+
+def compress_confusion_matrix(matrix: np.ndarray, use_zlib=True):
+    non_zero_indices = np.nonzero(matrix)
+    non_zero_matrix = [
+        {"i": int(i), "j": int(j), "v": int(matrix[i, j])}
+        for i, j in zip(*non_zero_indices)
```

Ruff (0.14.7)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)